dictionary.md

```diff
@@ -247,10 +295,16 @@ Catching a query from the user and redirecting it to the appropriate API functio
 
 ### Runner
 
-An executable written in one of the supported programming languages that starts the [sequence](#sequence) created by developer, inside the container controlled by the [supervisor](#supervisor)
+An executable written in one of the supported programming languages that starts the [Sequence](#sequence) created by developer, inside the container
 
 ## S
 
+### Scalability
+<!-- TODO -->
```

Comment:
I wouldn't explain it at that point. It's a very broad topic and we are not yet ready to support it.

Reply:
I agree. I removed it from the dictionary.